docs/reference/api/connectivity/bluetooth/Gap.md

```diff
@@ -1,11 +1,19 @@
 ## Gap
 
-[Add description here.]
+Generic Access Profile is the layer of the stack that handles connectivity tasks. This includes link establishment and termination, advertising and scanning.
+
+Devices with data to publish can use GAP to advertise. They can either include the data in the advertisement itself, inside the scan response, or leave it to be queried by a peer device after connection has been established.
```

Comment:
I'm not sure that *either* should be used in that sentence; the proposition it introduces are supposed to be exclusive.